pack/tde/tsh.json

```diff
@@ -50,6 +174,38 @@
 		"type_code": "event",
 		"xp": 0
 	},
+	{
+		"code": "06165",
+		"cost": 0,
+		"deck_limit": 2,
+		"faction_code": "survivor",
+		"flavor": "You pry it out of the corpse's hand.\nIt's not like he's going to complain.",
+		"illustrator": "Patrick McEvoy",
+		"name": "Scrounge for Supplies",
+		"pack_code": "tsh",
+		"position": 165,
+		"quantity": 2,
+		"text": "Choose a level 0 card in your discard pile. Add the chosen card to your hand.",
+		"traits": "Fortune.",
+		"type_code": "event",
+		"xp": 0
+	},
+	{
+		"code": "06166",
+		"cost": 0,
+		"deck_limit": 2,
+		"faction_code": "survivor",
+		"illustrator": "Lukas Banas",
+		"name": "Brute Force",
+		"pack_code": "tsh",
+		"position": 166,
+		"quantity": 2,
+		"skill_combat": 1,
+		"text": "Max 1 committed per skill test.\nWhile Brute Force is committed during a basic fight action, it gains [combat][combat] and the text: \"If this test is successful by 2 or more, this attack deals +2 damage.\"",
+		"traits": "Innate. Developed.",
+		"type_code": "skill",
+		"xp": 1
+	},
 	{
 		"code": "06167",
 		"cost": null,
```

Comment:
add  after 'faction code': 

"flavor": "\"Unassuming\" doesn't mean you're unprepared.",